src/protocolsupport/zplatform/impl/spigot/itemstack/SpigotItemStackWrapper.java

```diff
@@ -13,7 +13,7 @@
 public class SpigotItemStackWrapper extends ItemStackWrapper {
 
 	protected final ItemStack itemstack;
-	protected SpigotItemStackWrapper(ItemStack itemstack) {
+	public SpigotItemStackWrapper(ItemStack itemstack) {
```

Comment:
hell no

Reply:
oh yeah, that was from previously when i was trying to do things that didn't work, i'll undo that